lib/components/Widgets/WidgetContainer/index.tsx

```diff
@@ -1,48 +1,39 @@
+import { CardDescription } from "@/factorial-one"
 import {
   Card,
   CardContent,
-  CardDescription,
-  CardFooter,
   CardHeader,
+  CardInfo,
+  CardLink,
+  CardSubtitle,
   CardTitle,
 } from "@/ui/card"
-import { TrendingUp } from "lucide-react"
 import { forwardRef, ReactNode } from "react"
 
 export interface WidgetContainerProps {
   header: {
     title: string
-    description: string
-  }
-  footer?: {
-    trend: string
-    time: string
+    subtitle?: string
+    info?: string
+    link?: string
   }
+  description?: string
 }
 
 export const WidgetContainer = forwardRef<
   HTMLDivElement,
   WidgetContainerProps & { children: ReactNode }
->(({ header, footer, children }, ref) => (
-  <Card ref={ref}>
+>(({ header, description, children }, ref) => (
+  <Card ref={ref} className="min-h-[200px]">
     <CardHeader>
-      <CardTitle>{header.title}</CardTitle>
-      <CardDescription>{header.description}</CardDescription>
+      <div className="flex grow flex-row gap-1.5">
+        <CardTitle>{header.title}</CardTitle>
+        {header.subtitle && <CardSubtitle>{header.subtitle}</CardSubtitle>}
+        {header.info && <CardInfo>{header.info}</CardInfo>}
+      </div>
+      {header.link && <CardLink href={header.link} />}
```

Comment:
What if we tried making the whole card header clickable, not only the arrow?

Reply:
As we talked, we're opting for the arrow for the time being, but it's subject to change.